tests/mocks/stubCreateEthers.ts

```diff
@@ -0,0 +1,11 @@
+import { TokensController } from '../../src/assets/TokensController';
+
+const stubCreateEthers = (ctrl: TokensController, res: boolean) => {
```

Comment:
Instead of moving this out into a separate file, what do you think about having inside of the test file itself? In my opinion, when parts of tests are broken up across multiple files, unless there is a good reason for it — say, we have a complicated setup and it would bloat the test file to have everything in one file, or we need to use that setup across multiple files — it makes it difficult to understand the totality of a single test. In this case, I feel like it would be just as readable to have this directly in the test that uses it. What do you think?

Also, although we do prefer Jest mocks instead of Sinon, what do you think about continuing to use Sinon for this file until we make that conversion? Otherwise it might create confusion for someone reading this file.

Reply:
initially I had this helper fn in the test directly, but I needed to [use it in another test](https://github.com/MetaMask/controllers/pull/895/files#diff-35bf4fcbeee8c75b039b0098c4abab255947963772d6fa3e6ab05823039ea2b1R201) so I exported it, which gave a linting error about no exports from tests. 

Thats why I moved it into a seperate file. The alternative would be to dupe the code around in multiple tests which seems worse.

That said, if we prefer to dupe it, ill make the change.

Reply:
Yeah, if this were an implementation file then I would say it'd be fine to extract, but I think a little duplication is okay in test files, especially if it is 2 lines of code.